### DISCUSSION TITLE
Is there a way to create custom pages and forms?

### DISCUSSION THREAD

**Discussion by jackHedaya** [nocodb/nocodb#472] [Q&A] [answered]
I'm looking to self-host a custom dashboard/spreadsheet viewer. I need some custom buttons and pages but at the same time would love to use nocodb. Is there a way to add custom pages and access the vue underneath?

**Comment by o1lab** [comment, marked as answer]
We've a plan for a page designer. However, accessing vue beneath could be difficult.

Why would you like to access vue? would love to know the usecase

**Reply by jackHedaya** [reply]
A page designer might be all I need but I feel like there's a niche in the OS (and possibly even non-OS) market for customizable dashboards/data viewers. It would be very cool if there was a way to access the renderer below because then external components are at your disposal. Honestly, I'm sure for my use case, simply a page designer would work with the ability to send requests on form submissions and stuff like that. I guess something like this is somewhere between Nuxt and Noco and I know this is probably way out of Noco's use case/scope.

**Reply by o1lab** [reply]
I understand and have to agree this beyond our scope. And is an interesting project on its own :)
